### PR TITLE
Created FontResources and changed the PDFWithTimeNewRomanText.java to…

### PR DESCRIPTION
**Pull request by Nerd1000percent**
Created Test for the Multi font function.

### CODE REVIEW

**Comment by Nerd1000percent** [comment]
The subject name is wrong

**Review by A-Cain — CHANGES_REQUESTED** [review]
Firstly: pressing 2 for courier gives me a pdf that reads "This Text is in Times New Roman"
While I can tell it is in courier, it is still misleading.

Also, having to run the test twice to test both fonts is somewhat laborious for a test. It would be ideal if it either produced one pdf with both fonts present, or one pdf for each font. The first would best demonstrate the desired capability. 

Finally the test is not in junit format and thus will not run when a "run all tests" is called in the development environment. For the sake of continuity with other tests, I would ask that it be in junit format

**Comment by Nerd1000percent** [comment]
I have made the first 2 changes, not sure how to make a junit though.

**Comment by A-Cain** [comment]
> I have made the first 2 changes, not sure how to make a junit though.

the function that contains the test code should have 
@Test
in the line above it, and should be titled something like TestMultiFont or something similar. It should also end by asserting that each (or the one) created test file exists. There should be a comment telling reviewers that they need to manually look at the created test file to verify proper font use

**Comment by Nerd1000percent** [comment]
Awesome THANKS, I think its good now.

File: tools/src/test/java/org/apache/pdfbox/tools/MultiFontTest.java [hunk 1]
```diff
@@ -6,11 +6,14 @@
 import org.apache.pdfbox.pdmodel.PDPage;
 import org.apache.pdfbox.pdmodel.PDPageContentStream;
 import org.apache.pdfbox.pdmodel.font.PDType1Font;
+import org.junit.Test;
+
 import java.util.Scanner;
```
Comment (A-Cain) [inline tools/src/test/java/org/apache/pdfbox/tools/MultiFontTest.java:16]: The test function should not be public static void main

**Review by A-Cain — CHANGES_REQUESTED** [review]

**Comment by A-Cain** [comment]
Here is an example of one of my tests:

@Test
    public void testMain() throws IOException {
        AddImage add = new AddImage();
        add.loadFileandInitializeStream("../tools/src/main/java/org/apache/pdfbox/tools/AddImageResources/examplePDFWithText.pdf", 2);
        String imageString = "../tools/src/main/java/org/apache/pdfbox/tools/AddImageResources/pamcamke.jpg";
        PDImageXObject actual = add.createImage(imageString);
        add.writeImage("../tools/src/main/java/org/apache/pdfbox/tools/AddImageResources/examplePDFWithTextAndImage.pdf");
        File madeFile = new File("../tools/src/main/java/org/apache/pdfbox/tools/AddImageResources/examplePDFWithTextAndImage.pdf");
        assertTrue(madeFile.exists());
    }

note that it is not a main function, and that ti ends by asserting something that junit can use to determine a pass or fail

**Comment by Nerd1000percent** [comment]
Ok I see, I had to import that juint it wasn't working until I found the exact assert. I think its good now maybe.

**Comment by A-Cain** [comment]
Does the test pass on your machine? As of now it does not on mine because the file Courier.pdf is not created

**Comment by Nerd1000percent** [comment]
I see why,

**Comment by A-Cain** [comment]
Let me know when the issue is fixed and I will merge the request

**Comment by Nerd1000percent** [comment]
Ok I made some changes and it passes now.

**Comment by A-Cain** [comment]
I will tryst you that it works.

**Pull request merged**